mantidproject/mantid
# Inelastic Fortran scripts require compiling with gcc compiler

Fixes #15133

The f2py compiled Fortran scripts used in inelastic now compile with the gcc compiler instead of the Intel compiler.
The system test data and unit test data has been update to accommodate the change. 
Furthermore, the driver script for this `IndirectBayes` has been changed to improve execution time due to the way algorithm input properties are sometimes poorly handled in the log. A more permanent solution to this will be implemented in #15164 
# To Test
- Ensure System and Unit tests pass
- Ensure that the build environment is correctly configured when you open visual studio and the command prompt via the `.bat' files in the build directory
  - This can be checked by running the system test `ISISIndirectBayes` or by running Mantid and ensuring there are no warnings in the log about missing `.dll`s.
